webapp/tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": [
```

Comment:
Let's not commit this (even though this is autogenerated)

Reply:
which part exactly? is it the linting or the noEmit: false